getting-started/proc-add-source-control-rhdh-catalog.adoc

```diff
@@ -51,34 +80,97 @@ data:
 --
 
 . Click *Save*.
-. Create a key or value Secret file named `rhdh-secrets` using the environment variables as keys from the previous code snippet by following the steps below:
+. Navigate to the *Helm* tab and select *Upgrade*.
+. Under *Root Schema → Backstage Chart Schema → Backstage Parameters → Backstage container environment variables from existing Secrets*, input `github-secrets` as the value.
```

Comment:
Per IBM style guidance: Do not use "input" as a verb. Use type or enter as appropriate.